README.md

```diff
@@ -257,3 +256,118 @@ We can now scale Kibana and Fluentd back up to their previous state.  Since Flue
 was shut down and allowed to push its remaining records to ES in the previous
 steps it can now pick back up from where it left off with no loss of logs -- so long
 as the log files that were not read in are still available on the node.
+
+## EFK Health
+
+In order to determine that your EFK deployment is currently healthy and running,
```

Comment:
Not all of the e2e tests.  Only the check-EFK-running.sh, and possibly, the check-logs.sh should be run in a real deployment.  The other scripts such as check-curator.sh and check-es-copy.sh are _destructive_ and should not be run.  This includes the e2e-test.sh script, which will run check-curator and check-es-copy.

Perhaps the intention of the e2e-test.sh was that it could be run by anyone at any time?  If so, then we need another test driver script which will run all of the tests.